src/main/java/org/prebid/server/bidder/mobilefuse/MobilefuseBidder.java

```
@@ -75,6 +64,10 @@ private Imp modifyImp(Imp imp) {
         }
 
         final ExtImpMobilefuse impExt = parseImpExt(imp);
+        if (impExt == null) {
+            return null;
+        }
```

No need for this. If there is no `impExt` then imp will be filtered out for that bidder

---

Also, I see that in GO they fail on invalid `impExt` instead of ignoring errors

---

Thanks! I've removed the check as suggested.